api/FileReaderSync.json

```diff
@@ -20,7 +20,7 @@
             "version_added": "8"
           },
           "ie": {
-            "version_added": true
+            "version_added": false
```

Comment:
Per https://caniuse.com/filereadersync this was supported in IE10, and http://mdn-bcd-collector.appspot.com/tests/api/FileReaderSync is only checking for the interface object so false negatives are somewhat likely.

Can you check where the original data came from, and whether this was somehow supported in IE? Maybe the API shape was slightly different, or it was prefixed?

Reply:
It looks like there's an issue with getting exposure for `DedicatedWorker`, so because it was only testing in shared workers, which is not supported in IE, it was marking this as `false`.  Reverting.

Reply:
I see. I've filed https://github.com/foolip/mdn-bcd-collector/issues/815 to track this.